### ISSUE TITLE
Explain initial flatpak MAME setup

### ISSUE THREAD

**Issue by Abdull** [flathub/org.mamedev.MAME#54]
From the current description, it is impossible to set up a working instance.

E.g.
* how to set up directory structure for (existing) ROMs so that flatpak-based MAME can see them?
* how to set up controllers?

**Comment by jflatt** [comment]
There isn't a place in flathub for documentation.  Refer to the official mame site

**Comment by Abdull** [comment]
> There isn't a place in flathub for documentation.
Yes, there is; other flathub-hosted projects use their README.md. [*This* repository already has a README.md](https://github.com/flathub/org.mamedev.MAME/blob/master/README.md), so all it requires is adding a short section to explain the flatpak-specific aspects in setting up flatpak-MAME.

> Refer to the official mame site
The official MAME site says nothing about flatpak-specific aspects.

While my Bluetooth-attached game controller works (by now?) under Flatpak-MAME, I'm still stuck with figuring out how to set up the ROM directory. As you are a collaborator for *this* repository, I assume you should have at least once been able to successfully set up flatpak-MAME to load ROMs from your system's file system, and you should be aware that generally, directory setups are nontrivial between a flatpak instance and its "outside" world (due to overlays, permissions etc.). This includes aspects such as "does flatpak-MAME consider the host system's `$HOME/.mame` config directory, etc.?

In the flatpak-MAME GUI, *General Settings* > *Configure Folders* > *ROMs* > *ROMs Folders* lists a single entry, `/app/share/mame/roms`. Now this is of course the flatpak-MAME instance's own view of its own file system (explorable with `flatpak enter $(flatpak ps | grep mame | awk '{print $1}') bash`).

How to symlink an external ROM directory into this location? I guess you have done this at least once, maybe you can share your setup?
